CHANGELOG.md

```diff
@@ -106,6 +106,7 @@ Various fixes for Linux boot, More Chip/bringup examples, Chisel 3.5.6 bump
 * Support not instantiating tile reset/clock contorl features by @jerryz123 in https://github.com/ucb-bar/chipyard/pull/1459
 
 ### Fixed
+* Tiny documentation fix by @oharboe in https://github.com/ucb-bar/chipyard/pull/1744
```

Comment:
Can you remove this line? We actually autogen the CHANGELOG from the merged PRs whenever we bump the release version number, which we will do soonTM.